Source/WebCore/html/HTMLAttachmentElement.h

```diff
@@ -152,7 +152,7 @@ class HTMLAttachmentElement final : public HTMLElement {
     RefPtr<HTMLElement> m_saveButton;
     mutable RefPtr<DOMRectReadOnly> m_saveButtonClientRect;
 
-    bool m_needsWideLayoutIconRequest { false };
+    bool m_needsIconRequest { true };
```

Comment:
Can you elaborate on why the default value is being changed?

Reply:
It used to only be set to true from `ensureWideLayoutShadowTree()` (so only for wide-layout attachments), but now that we want to also set it for non-wide layout, we may as well just set it to true during construction.